gradle/SourceTemplate/app/build.gradle

```diff
@@ -0,0 +1,43 @@
+apply plugin: 'com.android.application'
+
+
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        // other classpath definitions here
+        classpath 'com.neenbedankt.gradle.plugins:android-apt:1.8'
+    }
+}
+
+android {
+    compileSdkVersion 25
```

Comment:
The template project is not part of the build, it is isolated so you can quickly check out whether the sources you write make any sense. Only the `src` dir of the template is actually copied to the main build.